packages/fetchai/skills/aries_faber/behaviours.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This package contains the behaviour for the aries_faber skill."""
+
+import json
+from typing import Dict
+
+from aea.configurations.base import PublicId
+from aea.skills.behaviours import OneShotBehaviour
+
+from packages.fetchai.protocols.http.message import HttpMessage
+from packages.fetchai.protocols.http.serialization import HttpSerializer
+
+
+HTTP_PROTOCOL_PUBLIC_ID = PublicId("fetchai", "http", "0.1.0")
```

Comment:
use HttpMessage.protocol_id instead, avoids versions being out of sync

Reply:
Done